
# HG changeset patch
# User Hannu Savolainen <hannu@opensound.com>
# Date 1202257182 -7200
# Node ID bc0a06e2e9bde5a503beb02d820125cf3ac46497
# Parent 1cb776eaa147ecdcc964299686b3a16ffebbec39
Changes to make libsalsa compatible with latest ALSA versions.

--- a/lib/libsalsa/seq.c	Tue Feb 05 14:42:35 2008 +0200
+++ b/lib/libsalsa/seq.c	Wed Feb 06 02:19:42 2008 +0200
@@ -1695,13 +1695,8 @@ const unsigned int snd_seq_event_types[2
     FIXED_EV (SND_SEQ_EVFLG_MESSAGE),
   [SND_SEQ_EVENT_PORT_SUBSCRIBED...SND_SEQ_EVENT_PORT_UNSUBSCRIBED] =
     FIXED_EV (SND_SEQ_EVFLG_CONNECTION),
-  [SND_SEQ_EVENT_SAMPLE...SND_SEQ_EVENT_SAMPLE_PRIVATE1] =
-    FIXED_EV (SND_SEQ_EVFLG_SAMPLE),
   [SND_SEQ_EVENT_USR0...SND_SEQ_EVENT_USR9] =
     FIXED_EV (SND_SEQ_EVFLG_RAW) | FIXED_EV (SND_SEQ_EVFLG_USERS),
-  [SND_SEQ_EVENT_INSTR_BEGIN...SND_SEQ_EVENT_INSTR_CHANGE] =
-    _SND_SEQ_TYPE (SND_SEQ_EVFLG_INSTR) |
-    _SND_SEQ_TYPE (SND_SEQ_EVFLG_VARUSR),
   [SND_SEQ_EVENT_SYSEX...SND_SEQ_EVENT_BOUNCE] =
     _SND_SEQ_TYPE (SND_SEQ_EVFLG_VARIABLE),
   [SND_SEQ_EVENT_USR_VAR0...SND_SEQ_EVENT_USR_VAR4] =
--- a/lib/libsalsa/seq_output.c	Tue Feb 05 14:42:35 2008 +0200
+++ b/lib/libsalsa/seq_output.c	Wed Feb 06 02:19:42 2008 +0200
@@ -254,33 +254,6 @@ convert_event (snd_seq_t * seq, snd_seq_
     case SND_SEQ_EVENT_PORT_UNSUBSCRIBED:
       dbg_printf ("\tSND_SEQ_EVENT_PORT_UNSUBSCRIBED\n");
       break;
-    case SND_SEQ_EVENT_SAMPLE:
-      dbg_printf ("\tSND_SEQ_EVENT_SAMPLE\n");
-      break;
-    case SND_SEQ_EVENT_SAMPLE_CLUSTER:
-      dbg_printf ("\tSND_SEQ_EVENT_SAMPLE_CLUSTER\n");
-      break;
-    case SND_SEQ_EVENT_SAMPLE_START:
-      dbg_printf ("\tSND_SEQ_EVENT_SAMPLE_START\n");
-      break;
-    case SND_SEQ_EVENT_SAMPLE_STOP:
-      dbg_printf ("\tSND_SEQ_EVENT_SAMPLE_STOP\n");
-      break;
-    case SND_SEQ_EVENT_SAMPLE_FREQ:
-      dbg_printf ("\tSND_SEQ_EVENT_SAMPLE_FREQ\n");
-      break;
-    case SND_SEQ_EVENT_SAMPLE_VOLUME:
-      dbg_printf ("\tSND_SEQ_EVENT_SAMPLE_VOLUME\n");
-      break;
-    case SND_SEQ_EVENT_SAMPLE_LOOP:
-      dbg_printf ("\tSND_SEQ_EVENT_SAMPLE_LOOP\n");
-      break;
-    case SND_SEQ_EVENT_SAMPLE_POSITION:
-      dbg_printf ("\tSND_SEQ_EVENT_SAMPLE_POSITION\n");
-      break;
-    case SND_SEQ_EVENT_SAMPLE_PRIVATE1:
-      dbg_printf ("\tSND_SEQ_EVENT_SAMPLE_PRIVATE1\n");
-      break;
     case SND_SEQ_EVENT_USR0:
       dbg_printf ("\tSND_SEQ_EVENT_USR0\n");
       break;
@@ -311,63 +284,6 @@ convert_event (snd_seq_t * seq, snd_seq_
     case SND_SEQ_EVENT_USR9:
       dbg_printf ("\tSND_SEQ_EVENT_USR9\n");
       break;
-    case SND_SEQ_EVENT_INSTR_BEGIN:
-      dbg_printf ("\tSND_SEQ_EVENT_INSTR_BEGIN\n");
-      break;
-    case SND_SEQ_EVENT_INSTR_END:
-      dbg_printf ("\tSND_SEQ_EVENT_INSTR_END\n");
-      break;
-    case SND_SEQ_EVENT_INSTR_INFO:
-      dbg_printf ("\tSND_SEQ_EVENT_INSTR_INFO\n");
-      break;
-    case SND_SEQ_EVENT_INSTR_INFO_RESULT:
-      dbg_printf ("\tSND_SEQ_EVENT_INSTR_INFO_RESULT\n");
-      break;
-    case SND_SEQ_EVENT_INSTR_FINFO:
-      dbg_printf ("\tSND_SEQ_EVENT_INSTR_FINFO\n");
-      break;
-    case SND_SEQ_EVENT_INSTR_FINFO_RESULT:
-      dbg_printf ("\tSND_SEQ_EVENT_INSTR_FINFO_RESULT\n");
-      break;
-    case SND_SEQ_EVENT_INSTR_RESET:
-      dbg_printf ("\tSND_SEQ_EVENT_INSTR_RESET\n");
-      break;
-    case SND_SEQ_EVENT_INSTR_STATUS:
-      dbg_printf ("\tSND_SEQ_EVENT_INSTR_STATUS\n");
-      break;
-    case SND_SEQ_EVENT_INSTR_STATUS_RESULT:
-      dbg_printf ("\tSND_SEQ_EVENT_INSTR_STATUS_RESULT\n");
-      break;
-    case SND_SEQ_EVENT_INSTR_PUT:
-      dbg_printf ("\tSND_SEQ_EVENT_INSTR_PUT\n");
-      break;
-    case SND_SEQ_EVENT_INSTR_GET:
-      dbg_printf ("\tSND_SEQ_EVENT_INSTR_GET\n");
-      break;
-    case SND_SEQ_EVENT_INSTR_GET_RESULT:
-      dbg_printf ("\tSND_SEQ_EVENT_INSTR_GET_RESULT\n");
-      break;
-    case SND_SEQ_EVENT_INSTR_FREE:
-      dbg_printf ("\tSND_SEQ_EVENT_INSTR_FREE\n");
-      break;
-    case SND_SEQ_EVENT_INSTR_LIST:
-      dbg_printf ("\tSND_SEQ_EVENT_INSTR_LIST\n");
-      break;
-    case SND_SEQ_EVENT_INSTR_LIST_RESULT:
-      dbg_printf ("\tSND_SEQ_EVENT_INSTR_LIST_RESULT\n");
-      break;
-    case SND_SEQ_EVENT_INSTR_CLUSTER:
-      dbg_printf ("\tSND_SEQ_EVENT_INSTR_CLUSTER\n");
-      break;
-    case SND_SEQ_EVENT_INSTR_CLUSTER_GET:
-      dbg_printf ("\tSND_SEQ_EVENT_INSTR_CLUSTER_GET\n");
-      break;
-    case SND_SEQ_EVENT_INSTR_CLUSTER_RESULT:
-      dbg_printf ("\tSND_SEQ_EVENT_INSTR_CLUSTER_RESULT\n");
-      break;
-    case SND_SEQ_EVENT_INSTR_CHANGE:
-      dbg_printf ("\tSND_SEQ_EVENT_INSTR_CHANGE\n");
-      break;
     case SND_SEQ_EVENT_SYSEX:
       dbg_printf ("\tSND_SEQ_EVENT_SYSEX\n");
       break;

